src/components/pages/UserParentalControl.tsx

```diff
@@ -0,0 +1,379 @@
+        }
+
+        setUnratedItems(itemsArr);
+
+        const blockUnratedItems = element?.current?.querySelector('.blockUnratedItems');
+        blockUnratedItems.dispatchEvent(new CustomEvent('create'));
+    }, []);
+
+    const loadBlockedTags = useCallback((tags) => {
+        setBlockedTags(tags);
+
+        const blockedTagsElem = element?.current?.querySelector('.blockedTags');
+
+        for (const btnDeleteTag of blockedTagsElem.querySelectorAll('.btnDeleteTag')) {
+            btnDeleteTag.addEventListener('click', function () {
+                const tag = btnDeleteTag.getAttribute('data-tag');
+                const newTags = tags.filter(function (t) {
+                    return t != tag;
+                });
+                loadBlockedTags(newTags);
+            });
+        }
+    }, []);
+
+    const renderAccessSchedule = useCallback((schedules) => {
+        setAccessSchedules(schedules);
+
+        const accessScheduleList = element?.current?.querySelector('.accessScheduleList');
+
+        for (const btnDelete of accessScheduleList.querySelectorAll('.btnDelete')) {
+            btnDelete.addEventListener('click', function () {
+                const index = parseInt(btnDelete.getAttribute('data-index'));
+                schedules.splice(index, 1);
+                const newindex = schedules.filter(function (i) {
+                    return i != index;
+                });
+                renderAccessSchedule(newindex);
+            });
+        }
+    }, []);
+
+    const loadUser = useCallback((user, allParentalRatings) => {
+        setUserName(user.Name);
+        LibraryMenu.setTitle(user.Name);
+        loadUnratedItems(user);
+
+        loadBlockedTags(user.Policy.BlockedTags);
+        populateRatings(allParentalRatings);
+        let ratingValue = '';
+
+        if (user.Policy.MaxParentalRating) {
+            for (let i = 0, length = allParentalRatings.length; i < length; i++) {
+                const rating = allParentalRatings[i];
+
+                if (user.Policy.MaxParentalRating >= rating.Value) {
+                    ratingValue = rating.Value;
+                }
+            }
+        }
+
+        element.current.querySelector('.selectMaxParentalRating').value = ratingValue;
+
+        if (user.Policy.IsAdministrator) {
+            element?.current?.querySelector('.accessScheduleSection').classList.add('hide');
+        } else {
+            element?.current?.querySelector('.accessScheduleSection').classList.remove('hide');
+        }
+        renderAccessSchedule(user.Policy.AccessSchedules || []);
+        loading.hide();
+    }, [loadBlockedTags, loadUnratedItems, populateRatings, renderAccessSchedule]);
+
+    const loadData = useCallback(() => {
+        loading.show();
+        const userId = appRouter.param('userId');
+        const promise1 = window.ApiClient.getUser(userId);
+        const promise2 = window.ApiClient.getParentalRatings();
+        // eslint-disable-next-line compat/compat
```

Comment:
You should be able to remove this now. I fixed the eslint configuration for typescript files yesterday.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/3072/commits/f5b69e0de3a0277b01738e5436357fd228abe16e